core/common/src/rpc/web/BentleyCloudRpcProtocol.ts

```diff
@@ -130,7 +130,7 @@ export abstract class BentleyCloudRpcProtocol extends WebAppRpcProtocol {
       if (key === "mode") {
         ++i;
       } else if (key === "context") {
```

Comment:
@swbsi like here

Reply:
This should be fine. It's just a variable rename. We aren't changing the /context/ part of the url...which would obviously break the orchestrator.

Reply:
I'll talk to the Orchestrator team about timeline for them accepting iTwinId as well as context.  Ideally we'd make this switch now, otherwise we'd have to wait until 4.0.

Reply:
> I'll talk to the Orchestrator team about timeline for them accepting iTwinId as well as context. Ideally we'd make this switch now, otherwise we'd have to wait until 4.0.

That would be independent of this PR, though.

We are not changing the “/context/<some value>” url-based ABI that we have with the orchestrator here.

Reply:
@calebmshafer @swbsi I really don't understand why we include both iTwinId and iModelId anyway. iModelId is globally unique.

Reply:
It had something to do with how the router directs requests. I don’t remember the details.

Reply:
@swbsi you're right, if we did update it that would go into a different PR.

@kabentley @swbsi that's a good question, I would assume it's mostly a permissions thing but you can do that directly using the iModel permissions... I'll bring this up in the discussion about iTwinid; if we actually need it at all.

Reply:
I talked to the team this morning and the reason for the Context Id is due to the permissions check.  If there aren't any specific iModel permissions defined, then they fall back to checking the permissions on the Context itself to determine whether they should route.

This workflow seems kind of strange and IMO not a reason to keep it there since you can always look up the Context/iTwin an iModel belongs to. For now we can leave this but I'll continue to follow up and see if we could just drop it before the 3.0 release.

However, none of this will affect this PR so going to resolve the conversation.